src/main/java/banana/Parser.java

```diff
@@ -0,0 +1,281 @@
+package banana;
+
+import java.util.ArrayList;
+import java.time.LocalDate;
+
+/**
+ * The Parser class makes sense
+ * of the user's input.
+ *
+ * @author: Ravi Ananya
+ **/
+
+public class Parser {
+
+    private String input;
+
+    /**
+     * Constructor for the Parser class.
+     *
+     * @param input the user input.
+     */
+    public Parser(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Gets the user input.
+     *
+     * @return the user input.
+     */
+    public String getInput() {
+        return input;
+    }
+
+    /**
+     * Prints the information.
+     *
+     * @param information display information.
+     */
+    public static void displayLabel(String information) {
+        String label =
+                "    ____________________________________________________________\n" +
+                "     " +
+                information + "\n    " +
+                "____________________________________________________________\n";
+        System.out.println(label);
+    }
+
+    /**
+     * Checks how to handle the user's
+     * input command.
+     *
+     * @param tasks the list of tasks.
+     * @throws DukeException if the input was invalid.
+     */
+    public void useInput(TaskList tasks) throws DukeException {
```

Comment:
Since there are many ways in which input can be used, should a more descriptive name be used for this method so as to further specify the purpose of this method? One possible suggestion is **parseInput**